packages/ag-grid-theme/css/_salt-ag-theme-icon-font.scss

```diff
@@ -0,0 +1,61 @@
+$icons: (
+  aggregation: "\e974",
+  arrows: "\e90e",
+  asc: "\e940",
+  cancel: "\e90e",
+  columns: "\e910",
+  copy: "\e90d",
+  csv: "\e93f",
+  desc: "\e93f",
+  filter: "\e922",
+  first: "\e934",
+  grip: "\e970",
+  group: "\e929",
+  last: "\e951",
+  menu: "\e969",
+  next: "\e950",
+  pin: "\e944",
+  pivot: "\e979",
+  previous: "\e933",
+  small-right: "\e950",
+  tick: "\e95b",
+  tree-closed: "\e950",
+  tree-open: "\e91b",
+);
+
+@font-face {
+  font-family: "jpmuitk-icons";
+  src: url("fonts/jpmuitk-icons.woff") format("woff");
+}
+
+@each $mode in "light", "dark" {
+  @each $name, $glyph in $icons {
+    .ag-theme-salt-high-#{$mode} .ag-icon-#{$name},
```

Comment:
We could but I was worry that higher complexity selectors would have a performance impact

Reply:
Good point. I never deeply investigated what the perf impact is of different selectors, and I do think the front end devs are disproportionately concerned with load time instead of run time perf (including me).